packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -218,15 +218,29 @@ class _TabView extends StatefulWidget {
 
 class _TabViewState extends State<_TabView> {
   List<Widget> tabs;
+  List<FocusScopeNode> tabFocusNodes;
 
   @override
   void initState() {
     super.initState();
     tabs = new List<Widget>(widget.tabNumber);
+    tabFocusNodes = new List<FocusScopeNode>.generate(
+      widget.tabNumber,
+      (_) => new FocusScopeNode(),
+    );
+  }
+
+  @override
+  void dispose() {
+    for (FocusScopeNode focusScopeNode in tabFocusNodes) {
+      focusScopeNode.detach();
+    }
+    super.dispose();
   }
 
   @override
   Widget build(BuildContext context) {
+    FocusScope.of(context).setFirstFocus(tabFocusNodes[widget.currentTabIndex]);
```

Comment:
It seems a bit weird to be calling this from build; for example during hot reload every one of these is going to fire and try to grab the focus.